### PR TITLE
Update microsoft-365-guest-settings.md

### LINKED ISSUE #12759: Incorrect Navigation guidance.
The following Navigation direction  found in the section:

https://learn.microsoft.com/en-us/microsoft-365/solutions/microsoft-365-guest-settings?view=o365-worldwide#site-file-and-folder-link-settings
is incorrect.

Navigation: SharePoint admin center > [Active sites]> select the site > Policies tab > Edit External sharing

It should be:
Navigation: SharePoint admin center > [Active sites]> select the site > **Settings** > Edit External sharing

[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 030743fd-0c22-025b-f190-f637a2772506
* Version Independent ID: fbded472-e3ea-4297-c527-810e117ea7ea
* Content: [Microsoft 365 guest sharing settings reference](https://learn.microsoft.com/en-us/microsoft-365/solutions/microsoft-365-guest-settings?view=o365-worldwide)
* Content Source: [microsoft-365/solutions/microsoft-365-guest-settings.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/solutions/microsoft-365-guest-settings.md)
* Service: **o365-solutions**
* GitHub Login: @MikePlumleyMSFT
* Microsoft Alias: **mikeplum**

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the navigation

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12759

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/solutions/microsoft-365-guest-settings.md [hunk 1]
```diff
@@ -244,7 +244,7 @@ If the site has a sensitivity label applied, that label may control the external
 
 You can set defaults for link type and permissions, and expiration settings for *Anyone* links for each site. When set at the site level, these settings override the organization-level settings. Note that if *Anyone* links are disabled at the organization level, *Anyone* will not be an available link type at the site level.
 
-**Navigation:** SharePoint admin center > <a href="https://go.microsoft.com/fwlink/?linkid=2185220" target="_blank">**Active sites**</a> > select the site > **Policies** tab > **Edit External sharing**
+**Navigation:** SharePoint admin center > <a href="https://go.microsoft.com/fwlink/?linkid=2185220" target="_blank">**Active sites**</a> > select the site > **Settings** > **Edit External sharing**
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/microsoft-365-guest-settings.md]:
```suggestion
**Navigation:** SharePoint admin center > <a href="https://go.microsoft.com/fwlink/?linkid=2185220" target="_blank">**Active sites**</a> > select the site > **Settings** > **Edit External sharing**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**